package.json

```diff
@@ -806,10 +813,22 @@
         "description": "Restarts a composition of containers",
         "category": "Docker"
       },
+      {
+        "command": "vscode-docker.compose.start",
+        "title": "Compose Restart",
+        "description": "Builds and starts a composition of containers",
+        "category": "Docker"
+      },
+      {
+        "command": "vscode-docker.compose.stop",
+        "title": "Compose Restart",
+        "description": "Stops a composition of containers (non-destructive)",
+        "category": "Docker"
+      },
       {
         "command": "vscode-docker.compose.up",
         "title": "Compose Up",
-        "description": "Starts a composition of containers",
+        "description": "Builds a composition of containers",
```

Comment:
Builds, (re)creates, starts, and attaches to containers for a service.

https://docs.docker.com/compose/reference/up/